lib/pinchflat_web/components/core_components.ex

```diff
@@ -644,7 +644,13 @@ defmodule PinchflatWeb.CoreComponents do
       <li :for={{k, v} <- @iterable_attributes} class="mb-2 w-2/3">
         <strong><%= k %>:</strong>
         <code class="inline-block text-sm font-mono text-gray p-0.5 mx-0.5">
-          <%= v %>
+          <%= if is_binary(v) and String.starts_with?(v, "http") do %>
+            <.link class="underline decoration-bodydark decoration-1 hover:decoration-white" href={v} target="_blank">
```

Comment:
I noticed that but didn't want to go importing random things without understanding the consequences. Looks like it works though.

Reply:
Overall, I find the attribute list serviceable, though it would be nice to resolve the index frequency to the "human-readable" equivalent (like in the drop-down list on the Source form).

If you're looking for a generic display solution, you could try replacing underscores with spaces and title casing the keys, though that wouldn't work well for things like "ID" or "UUID."

Displaying the attributes as a table, could help make it more readable.  For example:

![pinchflat_mockup](https://github.com/user-attachments/assets/8e0a08b7-6b82-436a-bf49-7f9fcfb73a2b)

![pinchflat_mockup_center_align](https://github.com/user-attachments/assets/1bf7ed66-847e-4abb-9ba4-b7737d520787)

![pinchflat_mockup_right_align](https://github.com/user-attachments/assets/ffdfa220-16a2-4a4e-91d6-1edbfe99fa48)